public/decapcms/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="robots" content="noindex" />
+    <title>Content Manager</title>
+    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+  </head>
+  <body>
+    <!-- Include the script that builds the page and powers Decap CMS -->
+    <script src="https://unpkg.com/decap-cms@^3.0.0/dist/decap-cms.js"></script>
```

Comment:
I'm assuming this is to provide an online editor for posts that will be added to the repo in markdown?

Reply:
Nah, completely unrelated. good spot.

Reply:
I didn't even realize that this was in.

Reply:
I'll check it out later.